packages/components/src/bookingTypeRenderer/__tests__/BookingTypeRenderer.test.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import * as React from 'react';
+import { render } from 'react-native-testing-library';
+import { View } from 'react-native';
+
+import BookingTypeRenderer from '../BookingTypeRenderer';
+
+const getComponent = (type: string) => (
+  <BookingTypeRenderer
+    type={type}
+    oneWayComponent={<View testID="one-way" />}
```

Comment:
Why not to use camelCase notation here?

I noticed that we use many variations of those flight types.
Right now you can find in the project at least following variations: `oneway, oneWay, OneWay`

Reply:
You are right, I will change it to camelCase

Reply:
🚀